lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -600,19 +600,26 @@ def exec_migration(conn, direction)
     super(conn, direction)
   end
 
-  def safely_acquire_lock_for_table(table, &block)
+  def safely_acquire_lock_for_table(table, mode: :access_exclusive, &block)
```

Comment:
Is it safe to skip the blocking transactions check if the mode is `access_share` or `row_share`?

Reply:
Should be addressed in https://github.com/braintree/pg_ha_migrations/pull/88/commits/cd6caf9f57e99319a31fccc87d18d4904ca0b618